demo-automation-java/src/test/java/com/example/tests/car_reviews_tests/CarReviewDetailPageConsumerReviewModuleTest.java

```diff
@@ -0,0 +1,94 @@
+package com.example.tests.car_reviews_tests;
+
+import com.example.core.datadriven.InfoCarReviewUnderTest;
+import com.example.pages.BasePage;
+import com.example.pages.car_reviews_pages.CarReviewDetailPageConsumerReviewModule;
+import com.example.pages.car_reviews_pages.CarReviewPageConsumerReviewModule;
+import com.example.pages.car_reviews_pages.ConsumerReviewPage;
+import com.example.pages.car_reviews_pages.WriteAReviewPage;
+import com.example.tests.BaseTest;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class CarReviewDetailPageConsumerReviewModuleTest extends BaseTest {
+    private void closePopUp() {
+        try {
+            keyword.click(keyword.waitForElementToBeClickable(driver.findElement(By.xpath("//button[@aria-label='Close']"))));
+        } catch (Exception e) {
+        }
+        try {
+            keyword.click(keyword.waitForElementToBeClickable(driver.findElement(By.cssSelector("#fsrFocusFirst"))));
+        } catch (Exception e) {
+        }
```

Comment:
In my opinion, I think you should write these methods in another class.

Reply:
I declare this function to close pop up for temporary, when another one handle this popup this function will be cleared